.github/workflows/release.yaml

```diff
@@ -272,6 +272,11 @@ jobs:
         run: |
           export "IMAGE=${{ matrix.url }}/${{ secrets[matrix.repository] }}"
           export "TAG=$(echo "${{ github.ref }}" | awk -F"/" '{print $3}')"
+          export "CHART_VERSION=$(echo "${{ github.ref_name }}" | sed 's/^v//')"
```

Comment:
instead of copying the content of the version.mk we should try to adapt the config.mk to properly support a tag imho

Reply:
You could make CHART_VERSION configurable e.g and set it via an env var

Reply:
I am having a look into that :+1: